Client/Frontend/Browser/TabManager.swift

```diff
@@ -81,6 +82,7 @@ class TabManager: NSObject {
   var selectedIndex: Int { return _selectedIndex }
   var tempTabs: [Tab]?
   private weak var rewards: BraveRewards?
+  var makeWalletProvider: ((Tab) -> (BraveWalletBraveWalletProvider, js: String)?)?
```

Comment:
Hmm this being used here a second time makes me think it may have been intentional. Is this a pattern that was deliberately chosen?

Reply:
Same as above, no additional action needed here